src/neynar-api/v2/client.ts

```diff
@@ -1803,7 +1840,7 @@ export class NeynarV2APIClient {
    * @param {'1d' | '7d' | '30d'} [timeWindow] - The time window for trending analysis. Options are '1d' (one day),
    *   '7d' (seven days), or '30d' (thirty days).
    * @param {Object} [options] - Optional parameters to tailor the request.
-   * @param {number} [options.limit=25] - The number of channel details to fetch per request. Defaults to 25, with a maximum allowable value of 100.
+   * @param {number} [options.limit=10] - The number of channel details to fetch per request. Defaults to 10, with a maximum allowable value of 25.
```

Comment:
Default is 25

Reply:
see here @Shreyaschorge https://github.com/neynarxyz/nodejs-sdk/pull/90/files#r1559907457